src/Components/Alert/Components/Filters/NotificationPreferences.tsx

```diff
@@ -0,0 +1,132 @@
+import {
+  Box,
+  Checkbox,
+  Message,
+  Skeleton,
+  SkeletonBox,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import { useAlertContext } from "Components/Alert/Hooks/useAlertContext"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { RouterLink } from "System/Router/RouterLink"
+import { FC, useEffect } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { NotificationPreferences_viewer$data } from "__generated__/NotificationPreferences_viewer.graphql"
+import { NotificationPreferencesQuery } from "__generated__/NotificationPreferencesQuery.graphql"
+import { AlertFormikValues } from "Components/Alert/Components/Steps/Details"
+import { useFormikContext } from "formik"
+
+interface NotificationPreferencesProps {
+  viewer?: NotificationPreferences_viewer$data
+}
+
+export const NotificationPreferences: FC<NotificationPreferencesProps> = ({
```

Comment:
Removed the component `[EmailPreferenceWarningMessage.tsx](https://github.com/artsy/force/pull/13192/files#diff-0f54fb26299f1257bcea26eff70538e9ff4d01d7bb6f07246300d8ab18f65c5a)` and created a new component with a query renderer for the notification settings.